microsoft-edge/devtools-guide-chromium/whats-new/2022/03/devtools.md

```diff
@@ -0,0 +1,209 @@
+Remember the section of the profile that you are looking at
+-->
+
+The unminified call stack in the **Performance** tool now shows those functions labeled with meaningful names: **invokeFunc (b)**, **executeAction (O)**, **endBatch (Xt)**, and **runReactions (bn)**:
+
+![The unminified call stack in the Performance tool.](../../media/2022/03/unminified-call-stack-performance-tool.png)
+<!-- Image 2: Unminified call stack
+Select the Unminify button
+Save the unminified profile (it'll be in the .json file format) to your Downloads folder
+Select Load profile and load the unminified perf profile you just downloaded
+Select the same section of the profile you looked at in the first image
+Draw red highlights around function names that should now be unminified. In my demo, 
+Xt => endBatch
+bn => runReactions
+i => finalDispatch
+ -->
+
+See also:
+
+* [Unminifying function names in DevTools performance profiles](https://blogs.windows.com/msedgedev/2022/02/03/unminifying-function-names-in-devtools-performance-profiles/) at the Microsoft Edge Blog.
+* [Unminify performance profiles (temp. PR link)](https://review.docs.microsoft.com/microsoft-edge/devtools-guide-chromium/evaluate-performance/unminify?branch=pr-en-us-1784).
+<!-- todo: update from review.dmc to live url -->
+
+
+<!-- ====================================================================== -->
+## Improved accessibility for Network Console and 3D View
+
+<!-- Title: Improvements for using assistive technology with DevTools -->
+<!-- Subtitle: Screen readers now announce better information in the Network Console and 3D View tools. -->
+
+In previous versions of Microsoft Edge, assistive technology announced incorrect information when providing a custom bearer token in the Network Console tool.  In Microsoft Edge version 99, this issue has been fixed.  When selecting the text box for providing a custom bearer token, assistive technology now announces "Token edit text Request View group".
+
+![The Network Console tool.](../../media/2022/03/network-console-tool.png)
+<!--
+1. Open DevTools.
+2. Press Ctrl+Shift+P and enter "Network Console".
+3. Open the Network Console tool.
+4. Select "Create a request".
+5. Select "Auth".
+6. Select "Bearer Token".
+7. Draw a red highlight around the "Token" edit textbox.
+8. If you can't see the "Token" edit textbox, drag the slider above "Hit 'Send' to issue the request" down to the bottom.
+-->
+
+In previous versions of Microsoft Edge, assistive technology would only announce the radio button text in the 3D View tool, without announcing the group name that the button belongs to.  In Microsoft Edge version 99, this has been fixed.  For example, when focus is on the **Use screen texture** radio button, assistive technology will now announce: "Color type radio group, Use screen texture, radio button, unselected".
```

Comment:
Maybe for this example about what assistive tech will announce, we should say "selected" since in the screenshot, that radio button is selected.
```suggestion
In previous versions of Microsoft Edge, assistive technology would only announce the radio button text in the 3D View tool, without announcing the group name that the button belongs to.  In Microsoft Edge version 99, this has been fixed.  For example, when focus is on the **Use screen texture** radio button, assistive technology will now announce: "Color type radio group, Use screen texture, radio button, selected".
```

Reply:
Your change makes more sense than what I copied from the work item.